src/integration-test/groovy/com/epseelon/TodoControllerSpec.groovy

```diff
@@ -1,20 +1,27 @@
 package com.epseelon
 
-import geb.spock.GebSpec
-import grails.plugins.rest.client.RestBuilder
+import grails.http.client.RxHttpClientBuilder
 import grails.test.mixin.integration.Integration
 import grails.transaction.Transactional
+import spock.lang.Shared
+import spock.lang.Specification
 
 /**
  * Created by sarbogast on 14/02/2017.
  */
 @Integration
 @Transactional
-class TodoControllerSpec extends GebSpec {
+class TodoControllerSpec extends Specification {
+
+    @Shared RxHttpClientBuilder rest = new RxHttpClientBuilder()
```

Comment:
@jeffbrown Is there some documentation somewhere about this RxHttpClientBuilder()? Because of course my real app is a little more complicated than this sample app, I need to add some Authorization header to the request and so on. And apparently it doesn't work like for RestBuilder.